synapseclient/core/retry.py

```diff
@@ -155,3 +175,9 @@ def _get_message(response):
     except (AttributeError, ValueError):
         # The response can be truncated. In which case, the message cannot be retrieved.
         return None
+
+
+class SynapseThrottling(Exception):
```

Comment:
Maybe rename it _SynapseThrottlingException (since it's internal to this module).